dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -526,7 +526,7 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
           data.method = DecodeMethod::NvjpegCuda;
           samples_single_.push_back(&data);
         }
-      } else if (!ParseNvjpeg2k(data, span<const uint8_t>(input_data, in_size))) {
+      } else if (crop_generator || !ParseNvjpeg2k(data, span<const uint8_t>(input_data, in_size))) {
```

Comment:
This looks good for now. Just an idea to be discussed with the team:
In case that ROI decoding is not an option with nvjpeg2k, should we decode the whole image with nvjpeg2k and launch the GPU slice kernel to produce the result instead?

Reply:
Added a comment to the schema docs,